Content.Server/_NF/M_Emp/M_EmpGridComponent.cs

```diff
@@ -0,0 +1,14 @@
+namespace Content.Server._NF.M_Emp
+{
+    /// <summary>
+    /// A grid spawned by a salvage magnet.
```

Comment:
make sure to update your comments to reflect the functions of your code hah

Reply:
Seem like a left over file, removed it.